document-store/src/main/java/org/hypertrace/core/documentstore/expression/impl/IdentifierExpression.java

```diff
@@ -23,10 +22,11 @@
 public class IdentifierExpression
     implements GroupingExpression, SelectingExpression, SortingExpression {
 
-  @NotBlank String name;
+  String name;
 
   public static IdentifierExpression of(final String name) {
-    return validateAndReturn(new IdentifierExpression(name));
+    Preconditions.checkArgument(!Strings.isNullOrEmpty(name), "name is null or emtpy");
```

Comment:
check for `string != null && string.isBlank()` instead.? this checks for `string != null && string.isEmpty()`

Reply:
I think you meant `string != null && !string.isBlank()`

Reply:
Done